### ISSUE TITLE
Quote choice is ignored if inside an interpolated string

### ISSUE THREAD

**Issue by jlabarbera11** [rubocop/rubocop#1181]
This is relative to the StringLiteral cop. When the enforced style for this is set to double_quotes, the string:

``` ruby
"hello#{hash['there']}"
```

should produce an error, since single quotes were used when not necessary. However, rubocop does not recognize it as such, since 'there' is part of an interpolated string, and thus ignored. This suggests that checking whether or not strings are a part of an ignored node is being done in the wrong place.
